Doc/tutorial/errors.rst

```diff
@@ -393,10 +393,12 @@ statement, whether an exception has occurred or not. When an exception has
 occurred in the :keyword:`!try` clause and has not been handled by an
 :keyword:`except` clause (or it has occurred in an :keyword:`!except` or
 :keyword:`!else` clause), it is re-raised after the :keyword:`finally` clause has
-been executed.  The :keyword:`!finally` clause is also executed "on the way out"
-when any other clause of the :keyword:`!try` statement is left via a
-:keyword:`break`, :keyword:`continue` or :keyword:`return` statement.  A more
-complicated example::
+been executed.  The :keyword:`!finally` clause is also executed penultimately
```

Comment:
Reading this for the first time still didn't make this clearer for me. Maybe it would be better to include an example such as https://discuss.python.org/t/pep-601-forbid-return-break-continue-breaking-out-of-finally/2239/3?